neuralpredictors/layers/cores/conv2d.py

```diff
@@ -542,3 +541,165 @@ def initialize(self):
         logger.warning(
             "Ignoring initialization since the parameters should be acquired from a pretrained model. If you want random weights, set pretrained = False."
         )
+
+
+class SE2dCore(Core2d, nn.Module):
+    def __init__(
+        self,
+        input_channels,
+        hidden_channels,
+        input_kern,
+        hidden_kern,
+        layers=3,
+        gamma_input=0.0,
+        skip=0,
+        final_nonlinearity=True,
+        bias=False,
+        momentum=0.1,
+        pad_input=True,
+        batch_norm=True,
+        hidden_dilation=1,
+        laplace_padding=None,
+        input_regularizer="LaplaceL2norm",
+        stack=None,
+        se_reduction=32,
+        n_se_blocks=0,
+        depth_separable=False,
+        attention_conv=False,
+        linear=False,
+    ):
+        """
+        Args:
+            input_channels:     Integer, number of input channels as in
+            hidden_channels:    Number of hidden channels (i.e feature maps) in each hidden layer
+            input_kern:     kernel size of the first layer (i.e. the input layer)
+            hidden_kern:    kernel size of each hidden layer's kernel
+            layers:         number of layers
+            gamma_input:    regularizer factor for the input weights (default: LaplaceL2, see neuralpredictors.regularizers)
+            skip:           Adds a skip connection
+            final_nonlinearity: Boolean, if true, appends an ELU layer after the last BatchNorm (if BN=True)
+            elu_xshift, elu_yshift: final_nonlinearity(x) = Elu(x - elu_xshift) + elu_yshift
+            bias:           Adds a bias layer.
+            momentum:       BN momentum
```

Comment:
`BN`? Does this mean batch-norm? In that case, what does that mean?

Reply:
Generally, if I read momentum, I'd expect it to be lr-related.

Reply:
It's an input to torch.nn.BatchNorm2d. I've clarified the doscstring.